reinstall.sh

```diff
@@ -10,8 +10,8 @@ echo 'Uninstalling stuff'
 ${PIP} uninstall -y nemo_toolkit
 ${PIP} uninstall -y sacrebleu
 
-# TODO: revert when 1.5.0 is out
-${PIP} uninstall -y pytorch-lightning
+# # TODO: revert when 1.5.0 is out
```

Comment:
maybe remove these comments since 1.5.0 is actually out?

Reply:
Yup, just wanted to CI to pass first